ingest/build-configs/ci/config.yaml

```diff
@@ -0,0 +1,6 @@
+custom_rules:
+  - "build-configs/ci/copy_example_data.smk"
+
+# Snakemake requires at least one top level key in a config file, so including
+# a bogus key here that should not be used anywhere in the Snakemake workflow
+bogus_ci_config: "bogus_ci_config"
```

Comment:
This bogus config is no longer needed since you've added the top `custom_rules` key.

Reply:
👍